docs/HowTo/Upgrade/Upgrade-Network.md

```diff
@@ -0,0 +1,32 @@
+description: Upgrade Pantheon     
+<!--- END of page meta data -->
+
+# Upgrading your Network 
+
+We recommend: 
+
+* Using an orchestration method (for example, Ansible or Chef) to keep all nodes in sync with your desired configuration. 
+* Storing your configuration under version control. 
+
+## Ansible 
+
+The [Pantheon Ansible role on Galaxy](https://galaxy.ansible.com/pegasyseng/pantheon) can be used directly
+or customised to suit your needs. Upgrade by running the play. The play: 
```

Comment:
@joshuafernandes - not sure if I have wording right here around 'the play'

Reply:
Maybe "Upgrade the pantheon version on nodes by running the play with the new version. See the Readme on Galaxy for details"?